src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import core.basesyntax.constant.Color;
+import core.basesyntax.producers.ColorProducer;
+
+public abstract class Figure implements Drawable {
```

Comment:
you figure should have field - color (it's a state for each object )

Reply:
done